doc/nrf/links.txt

```diff
@@ -107,11 +107,13 @@
 .. _`DCL Quick Start Guide`: https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/docs/quickStartGuide.adoc
 
 .. _`nRF Cloud JSON protocol schemas`: https://github.com/nRFCloud/application-protocols
+.. _`device_credentials_installer.py`: https://github.com/nRFCloud/utils/blob/master/python/modem-firmware-1.3%2B/device_credentials_installer.py
+.. _`nrf_cloud_provision.py`: https://github.com/nRFCloud/utils/blob/master/python/modem-firmware-1.3%2B/README.md#nrf-cloud-device-provisioning
+.. _`nRF Cloud Utilities documentation`: https://github.com/nRFCloud/utils/blob/master/python/modem-firmware-1.3%2B/README.md
```

Comment:
I agree these are rather stuff from github.com. However, my thinking was that they are closely related to nRF Cloud and thus, ended up adding them there.

Reply:
Yeah, I think we need to think a bit how to organize these links better, but until we have a new way of better sorting them (subsections?), I'd stick to current order criteria, hence the PR.